srcpkgs/pinephone-kernel/patches/smsc95xx-mac.patch

```diff
@@ -0,0 +1,95 @@
+From e36d607d56dc5c0cbf2cb600e7686b559ea77b44 Mon Sep 17 00:00:00 2001
+From: popcornmix <popcornmix@gmail.com>
```

Comment:
Does pinephone have `smsc95xx-mac` device?
Otherwise, I don't find anything obviously wrong, and #24307 is merged, I'll merge this after building.

Reply:
I kept it from the 5.7 patches since it's for a USB networking device. Someone could potentially use it thru a USB hub, unless I misunderstood what the `/net/usb` category is for. If it is a misunderstanding I'll delete it.

Reply:
I'll keep this in for now as it does seem that these are for USB attached devices, but I can still remove it if it's the right thing to do. https://github.com/torvalds/linux/blob/master/drivers/net/usb/Kconfig

Reply:
From what I understand, it's a problem when booting over NFS in the past,
the patch was rejected upstream [1], since then, firmware people had fixed the firmware [2]


1: https://netdev.vger.kernel.narkive.com/Fi2roLel/patch-smsc95xx-add-module-parameter-to-override-mac-address
2: https://github.com/raspberrypi/firmware/issues/613#issuecomment-282681283

Reply:
Makes sense, I'll drop it now